src/test/ui/consts/ptr_is_null.rs

```diff
@@ -0,0 +1,16 @@
+// compile-flags: --crate-type=lib
+// check-pass
```

Comment:
Are you sure `check-pass` is enough for this?

Reply:
considering that it did cause stderr output when I had the asserts inverted, yes. 

These constants are evaluated by the lint that used to be `const_err`, and `check` runs all lints.